src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx

```diff
@@ -171,3 +176,137 @@ export const useGetTableMenuProps = () =>
       ],
     };
   }, []);
+
+export const useBalanceTable = () => {
+  const { colony } = useColonyContext();
+  const selectedTeam = useGetSelectedTeamFilter();
+  const { balances, nativeToken } = colony || {};
+  const [searchValue, setSearchValue] = useState('');
+  const [value, setValue] = useState<Partial<BalanceTableFilters>>({
```

Comment:
is there an option for better types? what are all the possible values? can we use some attribute names, can the filter state be null? `Partial` is quite broad